src/cli.js

```diff
@@ -1,18 +1,9 @@
 const yargs = require('yargs');
-const supportedLocales = require('./util/supportedLocales');
-const defaultLocales = process.env.NODE_ENV === 'production'
-	? supportedLocales
-	: supportedLocales.slice(0, 1); // default to top locale in dev
 
 const run = () => {
 	const argv = yargs
 		.commandDir('commands') // commands are in the './commands/' dir
 		.demandCommand() // require a command to be used - no default behavior
-		.array('locales') // treat locales as array, always
```

Comment:
this was intentional - I want it to be an option that can be applied to both `run` and `build` commands